storybookjs/storybook
# UI: Keep mobile drawer open on component selection

Closes https://github.com/storybookjs/storybook/issues/24255
Closes https://github.com/storybookjs/storybook/issues/24256

## What I did

On the new mobile navigation we debated on the best way to close the main navigation when selected a story. We decided that the drawer will only close if a story is selected, not when a component is toggled. The behaviour remain unchanged on desktop.

I also improved a few styles on the sidebar as well as the about section on mobile

## Checklist for Contributors

### Testing



#### The changes in this PR are covered in the following automated tests:

#### Manual testing

- Run Storybook UI locally on your phone
- Open the main navigation
- Press on a component
- It should open the list of stories but not select any stories

## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
